### PR TITLE
(bugfix): Adapt to removal of xlocale.h

### PR DESCRIPTION
**Pull request by fsbruva**
Patch ICU to permit compilation on some platforms with updated glibc. Remove
instruction regarding symlink from README.

### CODE REVIEW

File: CPAN/buildme.sh [hunk 1]
```diff
@@ -435,8 +435,10 @@ if [ "$PERL_BIN" = "" -o "$CUSTOM_PERL" != "" ]; then
 
 fi
```
Comment (mherger) [inline CPAN/buildme.sh]: This is an unrelated change and doesn't belong here. Please submit its own change request with information about why you do this change.

Comment (fsbruva) [reply CPAN/buildme.sh]: @mherger This branch is reset.

**Comment by fsbruva** [comment]
Can you please check my branch to see if the build works for you? https://github.com/fsbruva/slimserver-vendor/commit/7ef5559ebe215f1d17905acdbb56e52e198e7f5f
